namespaces/Representor.md

```diff
@@ -0,0 +1,47 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Element)
+
+The Resource represents a Hypermedia Resource, with its available transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `relation` - (string) - An optional relation to this resource, applicable when embedded in another resource.
+- `content` (object)
```

Comment:
You know, what about going for the hash and dealing with resolving some of these conflicts later?